Assets/Fungus/Scripts/Editor/HierarchyIcons.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEditor;
+using System.Linq;
+
+namespace Fungus
+{
+    /// <summary>
+    /// Static class that hooks into the hierachy changed and item gui callbacks to put
+    /// a fungus icon infront of all GOs that have a flowchart on them
+    /// 
+    /// Reference; http://answers.unity3d.com/questions/431952/how-to-show-an-icon-in-hierarchy-view.html
+    /// </summary>
+    [InitializeOnLoad]
+    public class HierarchyIcons
+    {
+        // the fungus mushroom icon
+        static Texture2D textureIcon;
+
+        //sorted list of the GO instance IDs that have flowcharts on them
+        static List<int> flowchartIDs = new List<int>();
+
+        static HierarchyIcons()
+        {
+            //please don't move the fungus icon :(
+            textureIcon = AssetDatabase.LoadAssetAtPath("Assets/Fungus/Textures/ScriptIcon.png", typeof(Texture2D)) as Texture2D;
```

Comment:
You should use the FungusEditorResources system to access ScriptIcon.ng instead of loading it via AssetDatabase. If you do it that way then the icon will load even if the Fungus folder has been moved around.